distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1361,7 +1361,7 @@ type Table
 
     ## GROUP Standard.Base.Calculations
        ICON join
-       Replaces values in this table by values from a lookup table.
+       Merges this table with a lookup table
        New values are looked up in the lookup table based on the `key_columns`.
        Columns found in the lookup table values are replaced by values from the
        lookup. Columns not found are left unchanged.
```

Comment:
This is not from your changes, but I just realized that this sentence probably needs some rephrasing.
I don't think I understand the part "Columns found in the lookup table values" - a column is not really found in a value, I think there was some error here (I might have wrote it 😅). We probably should rephrase it to be more understandable.

Reply:
Rephrased to:

`       Columns that exist in the lookup table where a match was found are
       replaced by values from the lookup table. Columns not found are left
       unchanged.`